cps/web.py

```diff
@@ -21,6 +21,9 @@
 import base64
 from sqlalchemy.sql import *
 import json
+from wand.image import Image
```

Comment:
I think we can do without the extra config option.
Just catch the import error and set a variable and later use a generic cover if ImageMagick is not  installed:

```Python
try:
    from wand.image import Image
except ImportError, e:
    print "Could not import wand.image:"
    print e
    use_generic_pdf_cpver = True
```